src/ui/components/shared/Login/Login.tsx

```diff
@@ -175,21 +175,33 @@ function AuthError({ error }: { error: any }) {
     return null;
   }
 
-  if (message === "Invalid state") {
-    // This is usually caused by waiting too long to go through the auth process
-    // and can be fixed by trying again.
-    message = "Your login session expired. Please try logging in again.";
-  } else {
-    // We want to capture any other error so we can investigate further.
+  if (message !== "INVALID_STATE" && message !== "Invalid state") {
```

Comment:
Where do these message strings come from (e.g. "INVALID_STATE")? I'm curious because usually this type of message has some kind of constant we could import/reference from the package we're using (rather than hard-code strings).

Side note: Can you add a comment explaining that "Invalid state" comes from Auth0 and that's why we also check for this different looking message here? Maybe we could link to the docs, e.g. https://auth0.com/docs/customize/integrations/cms/wordpress-plugin/troubleshoot-wordpress-plugin-invalid-state-errors

Reply:
Except "Invalid state", we define every other error type in Auth0 rules.

Adding comment.